paywall/src/components/Paywall.css

```diff
@@ -14,7 +14,7 @@ body.big {
   overflow: initial;
 }
 
-@media only screen and (min-width: 135px) and (max-width: 763px) {
+@media only screen and (min-width: 257px) and (max-width: 763px) {
```

Comment:
Hum?

Reply:
Ah, I see. The addition of the duration info expands the total width of the flag, so the media queries have to account for it.

Reply:
yeah it took me quite a while to figure out how to handle this issue, because literally nothing else works except passing in the screen width using postmessage and doing all styling in javascript. To summarize my reaction to that possibility with "eek" would be an understatement. I much prefer this unintuitive 10-character performant solution :)